quote.py

```diff
@@ -1,10 +1,25 @@
+import re
 from paragraph import Paragraph
-
 class QuoteParagraph(Paragraph):
     """Element decorated by '>'.
     """
     def __init__(self, content):
         super().__init__(content)
 
     def render(self) -> str:
-        return "<blockquote> {} </blockquote>".format(self.inner_text())
+        return "<blockquote>{}</blockquote>".format(self.inner_text())
+
+    @staticmethod
+    def parse(content) -> (int, int, 'QuoteParagraph'):
+        """Parse a quote content, and return (begin, end, QuoteParagraph) if parseable,
+        or (-1, -1, None) that no such a element
+        """
+        pattern = r'^[ ]*\>[ ]*(.*)$'
```

Comment:
Should we consider cases with multiple `>`?
For example, `>>a`, the content is `a`. In this regex it will be `>a`. It seems wrong for nested cases.

Reply:
each parse will only take away one '>', so we can support nested '>>'. its intended.

Reply:
This implementation is fine. `> > a` will become `<blockquote><blockquote>a</></>`, so we need two level of QuoteParagraph embeded.